_rules/status-text-has-aria-live-mogq50.md

```diff
@@ -0,0 +1,525 @@
+---
+id: mogq50
+name: 'Status text update has `aria-live` property'
+rule_type: atomic
+description: |
+  This rule checks that any text update that meets the definition of a status message has either "polite" or "assertive" `aria-live` property.
+accessibility_requirements:
+  wcag21:4.1.3: # Status Messages (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:ARIA19: # Using ARIA role=alert or Live Regions to Identify Errors
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:ARIA22: # Using role=status to present status messages
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:ARIA23: # Using role=log to identify sequential information updates
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility Tree
+acknowledgments:
+  authors:
+    - aron janecki
+    - jean-yves moyen
+---
+
+## Applicability
+
+This rule applies to any [HTML element][] that has a [text node][] as a [descendant][] in the [flat tree][] if all of the following are true:
+
+- **available**: the element is [included in the accessibility tree][]; and
+- **change**: the element or its [descendant][] [text node][] value is an [event originated change in the content][].
+
+## Expectation 1
+
+There exists at least one test target with an implicit or explicit `aria-live` value of "assertive" or "polite" for [event originated changes in the content][event originated change in the content] that communicate the same message.
```

Comment:
This doesn't quite follow the rules format. An expectation must be true/false for each test target. This makes it seem like the expectation is true only once for whatever context it is in. Presumably per HTML page? But then what about EPUB, or pages with iframes?